doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -169,6 +169,115 @@ Create OpenLayers application
 
 These tiles are being rendered by the OpenLayers client.
 
+Create OpenLayers application - WMS Vector Tiles
+------------------------------------------------
+
+.. note::   Vector tiles requested with WMS allows to retrieve non-cached vector tiles (server side), setting the ``tiled=false`` parameter on the ``getMap`` request. This setting could be particularly useful when serving fast changing source data that should constantly be kept up-to-date for display.
+
+#. Create a ``www/wms-vectortiles`` directory inside your GeoServer Data Directory.
+
+#. Download the `latest version of OpenLayers <http://openlayers.org/download/>`_.
```

Comment:
I see two downloads, site and package... I guess you mean the "package" one?
I see "ol.css" in the root of the zip, "ol.js" in the "dist" zip, don't see a ``ol-debug.js`` though.
This part of the instructions should contain a bit more detail.

Reply:
@aaime, yes, I copied and pasted it from the previous part. Both should be rewritten since the downloadable `.zip` of OpenLayers has changed structure

Reply:
Then please do so :-D

Reply:
Done, corrected with the right paths to the files